workflow-templates/im-build-dotnet-ci.yml

```diff
@@ -339,16 +339,9 @@ jobs:
       #     read-pkg-token: ${{ secrets.READ_PKG_TOKEN }} # This is an org-level secret
       #     orgs: 'im-client,im-enrollment,im-practices' # TODO:  Verify list of orgs packages will be pulled from
 
-      # TODO:  If there are multiple deployables, duplicate this step once per deployable project and update the PROJECT_ROOT, PUBLISHED_APP and DEPLOY_ZIP variables with the updated names.
-      - name: Build and Publish
-        working-directory: ${{ env.PROJECT_ROOT }}
-        run: |
-          dotnet build --configuration Release
-          dotnet publish -c Release -o ./${{ env.PUBLISHED_APP }} --no-restore
-          (cd ${{ env.PUBLISHED_APP }} && zip -r ../${{ env.DEPLOY_ZIP }} .)
-
       # TODO: If you have multiple deployables in the repo and each deployable has it's own tag:
       #       1 - Duplicate this step once per deployable
+      #       2 - Duplicate the following Build and Publish step, changing it to build one deployable at a time instead of the whole solution.
```

Comment:
Since this is the new #2, the subsequent items need to be updated.

Reply:
oops. yep. will do.